doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -123,6 +123,17 @@ Enhanced ShockBurst (ESB)
 
 |no_changes_yet_note|
 
+* Added:
+
+  * The `optimize_txen_delay` feature that reduces radio TXEN delay from 130 µs to 40 µs.
+  * The :kconfig:option:`ESB_NEVER_DISABLE_TX` Kconfig option as an experimental feature that enables the radio peripheral to remain in TXIDLE state instead of TXDISABLE when transmission is pending.
```

Comment:
```suggestion
  * The :kconfig:option:`CONFIG_ESB_NEVER_DISABLE_TX` Kconfig option as an experimental feature that enables the radio peripheral to remain in TXIDLE state instead of TXDISABLE when transmission is pending.
```